en/closedbeta.md

```diff
@@ -1,42 +1,42 @@
 #What is findx closed beta access?
-Beta access to findx means that you will experience findx  while it is being build - The Privacore team knows there is a lot of work to do before we are ready to a public release, but we want to let users follow the progress while we are building the new real internet search engine from Europe.
+Beta access to findx means that you will experience findx while it's being built. The Privacore team knows there is a lot of work to do before we are ready for a public release, but we want to let users follow our progress while we are building the new real internet search engine from Europe.[OR "EUROPE'S NEW INTERNET SEARCH ENGINE."]
```

Comment:
It is an important message that it is a "search engine from Europe" rather than Europe's new search engine" - The first appeals broader